### DISCUSSION TITLE
Customize left/label color

### DISCUSSION THREAD

**Discussion by rarkins** [badges/shields#5751] [Q&A] [answered]
Is it possible to customize the left side of badges too, or only the right? e.g. Can I replace the default grey on the left with black or blue?

![image](https://user-images.githubusercontent.com/6311784/96561200-8ab36c00-12bf-11eb-8afa-0819189de82d.png)

^^ replacing the grey behind `hello` with blue, for example

**Comment by chris48s** [comment, marked as answer]
Yep - use the `labelColor` param :)

![](http://img.shields.io/badge/hello-world-green?labelColor=pink) http://img.shields.io/badge/hello-world-green?labelColor=pink
